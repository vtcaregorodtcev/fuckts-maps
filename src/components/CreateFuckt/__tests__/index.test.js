import React from 'react';
import renderer from 'react-test-renderer';
import { CreateFucktInput } from '../index';

it('renders correctly', () => {
  const tree = renderer.create(<CreateFucktInput />).toJSON();
  expect(tree).toMatchSnapshot();
});
