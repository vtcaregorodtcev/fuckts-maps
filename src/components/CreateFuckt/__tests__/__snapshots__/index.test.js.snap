// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="ant-input-search pov-page__create-input ant-input-search-enter-button ant-input-search-large ant-input-group-wrapper ant-input-group-wrapper-lg"
>
  <span
    className="ant-input-wrapper ant-input-group"
  >
    <input
      className="ant-input ant-input-lg"
      data-cy-create-fuckt={true}
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Введи свой факт"
      style={null}
      type="text"
      value=""
    />
    <span
      className="ant-input-group-addon"
    >
      <button
        className="ant-btn ant-input-search-button ant-btn-primary ant-btn-lg"
        onClick={[Function]}
        type="button"
      >
        <span>
          Добавить
        </span>
      </button>
    </span>
  </span>
</span>
`;
